src/alpha-lab/nerfs.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Create 3D Models of your environment with reference image mapper and nerfstudio
+description: "Nerfing your eye tracking recordings: use nerfstudio to create 3D models of your environment, and plot gaze in 3D."
+permalink: /alpha-lab/nerfs
+tags: [Pupil Invisible, Neon, Cloud]
+---
+<!-- <head> <script type="module" src="https://ajax.googleapis.com/ajax/libs/model-viewer/3.0.1/model-viewer.min.js"></script> </head> -->
+# Create 3D Models of Your Environment with Reference Image Mapper and NerfStudio
+
+<TagLinks />
+<Youtube src="ZSWl8qQcQk0"/>
+
+::: danger
+🕰️ - *Great Scott! This content is highly experimental. Using it will take you on a wild ride into the future, but beware - you'll be going solo. Consider yourself warned!* 🎢
+:::
+
+If you watched the accompanying video, you have witnessed a 3D reconstruction of an environment based on an eye tracking recording. In essence, we have explored how to augment the output of our [reference image mapper enrichment](/invisible/enrichments/reference-image-mapper/) to show a third person view of an eye tracking recording. Other wordly, huh?
+
+A third person view allows you to see more of the environment and how your partipant explores and visually interacts with it. Let's break it down – in the video, the green points denote where the “user’s head” was during the recording, while the yellow line illustrates a gaze ray from the head to the object that's looked at. You can also see a 3D heat-map showing which areas attracted attention.
+
+## A 3D view?
+
+We do perceive the world around us in 3D and for many years, we have tried to capture and reconstruct this 3D world using photogrammetry or special cameras. This approach was traditionally quite expensive and/or required specific types of cameras capable of recording depth and localising themselves in the scene.
+
+But nowadays, thanks to recent advances in deep learning, we have an easier way to reconstruct and build 3D environments. Isn't that exciting?!
+
+## What are NeRFs and how do they work?
+
+That advance we are talking about are [Neural Radiance Fields](https://arxiv.org/pdf/2003.08934.pdf) or NeRFs 🔫. NeRFs are a relatively novel method that use deep neural networks. They learn how light and colour vary based on the viewer's location and direction. So, by providing this tool with a set of images of a scene from different angles, it can generate novel views that were never actually captured by the camera.
+
+With this technique, we can create high-fidelity and photorealistic 3D models that can be used for various applications such as virtual reality, robotics, urban mapping, or in our case, to understand how the wearer was moving and looking in their environment. This approach doesn't need endless pictures of the environment, just a set of frames and camera poses (where the camera was located and where it pointed to).
```

Comment:
What's the difference between "endless pictures of the environment" to "just a set of frames and camera poses"? It seems in a sense, the second is more (as it includes poses).